ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/versioning/AppVersionConfig.java

```diff
@@ -0,0 +1,14 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.versioning;
+
+import org.springframework.boot.info.BuildProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AppVersionConfig {
+
+	@Bean
+	public String appVersion(BuildProperties buildProperties) {
+		return buildProperties.getVersion();
+	}
+}
```

Comment:
To me, this is too bloated.. Why not just use the build properties where needed? this created extra beans with no usage.